virt/virt-4-13-release-notes.adoc

```diff
@@ -59,6 +59,7 @@ All VM templates that are included with {VirtProductName} now use this machine t
 //** To learn more, see xref .. virt/upgrading-virt.adoc #virt-rhel-9_upgrading-virt[{VirtProductName} on {op-system-base} 9].
 
 //CNV-22041 Release note: NEW Retrieve a portable VM definition (yaml) when exporting a VM or VMSnapshot
+* You can now obtain the `VirtualMachine`, `DataVolume`, and `ConfigMap` manifests from the export server after you export a VM or snapshot. Learn more about xref:../virt/virtual_machines/virt-exporting-vms.adoc#virt-accessing-exported-vm-manifests_virt-exporting-vms[accessing exported VM manifests].
```

Comment:
Totally fine if you use this wording often other places, but the "Learn more about" sounds slightly off to me. I think people more often do the "For more information, see <link>."  wording